core/assets/bundles/bundle_pl.properties

```diff
@@ -159,15 +159,15 @@ save.rename = Zmień nazwę
 save.rename.text = Nowa nazwa:
 selectslot = Wybierz zapis.
 slot = [accent]Slot {0}
-editmessage = Edit Message
+editmessage = Edytuj wiadomość
 save.corrupted = [accent]Zapis gry jest uszkodzony lub nieprawidłowy! Jeżeli aktualizowałeś grę, najprawdopodobniej jest to zmiana w formacie zapisu i [scarlet]nie jest[] to błąd.
 empty = <pusto>
 on = Włączone
 off = Wyłączone
 save.autosave = Autozapis: {0}
 save.map = Mapa: {0}
 save.wave = Fala {0}
-save.mode = Gamemode: {0}
+save.mode = Tryb: {0}
```

Comment:
Maybe "Tryb gry"?

Reply:
All those change already exists in #765...